content/agent/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Agent"
+weight: 7
+logo: /images/opencensus-logo.png
+---
+
+![](/images/agent-server.png)
+
+- [Introduction](#introduction)
+- [Benefits](#benefits)
+- [Getting started](#getting-started)
+- [References](#references)
+
+### Introduction
+The OpenCensus Agent is a daemon that allows polyglot deployments of OpenCensus to use centralized
+exporter implementations. Instead of traditionally taking down and configuring OpenCensus exporters
+per language library and for every single application, with the OpenCensus Agent, one just has to singly
+enable the OpenCensus Agent exporter for their target language and it will deliver stats, metrics and traces
+to the centralized point before they are exported to the target backends.
+
+By default, ocagent runs on TCP port `55678` and receives traffic from:
+
+* HTTP/2 clients with Protobuf messages
+* HTTP/1 clients that use the [grpc-web-gateway](https://github.com/grpc-ecosystem/grpc-gateway)
+
+ocagent is written in the [Go programming language](https://golang.org/), it is cross platform, self-monitored and receives traffic
+from any application that supports the [ocagent protocol](https://github.com/census-instrumentation/opencensus-proto/tree/master/src/opencensus/proto/agent) regardless of the programming language and deployment.
```

Comment:
More precisely, Agent can receive traffic from applications instrumented with libraries that the receivers supported, like Zipkin/Jaeger/Prometheus. Supporting the ocagent protocol shouldn't be a requirement here.

Reply:
That can be added about receivers(that's a different point), but the point here is that if your language/library supports the agent protocol, then you can use it with the agent.

Reply:
Also, I plan on adding a section for receivers hence why I won't be providing a URL :)